plugins/services/src/js/containers/pod-debug/PodDebugContainer.js

```diff
@@ -41,8 +40,7 @@ class PodDebugTabView extends React.Component {
     const {pod} = this.props;
     const queue = pod.getQueue();
 
-    if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(pod)
```

Comment:
We are still using it in the `ServiceStatusWarning` component, here: https://github.com/dcos/dcos-ui/blob/6709773bd605e91d19c7cc3d59221168b444618f/plugins/services/src/js/components/ServiceStatusWarning.js#L16